deploy/deploy-k8s.yml

```diff
@@ -9,3 +29,13 @@
     kubeconfig_localhost: true
```

Comment:
This variable makes kubespray sync a copy of the kubeconfig to `{artifacts_dir}/admin.conf`. `artifacts_dir` defaults to '<Ansible inventor dir>/artifacts`, which is `.vagrant/provisioners/ansible/inventory/artifacts` with the Vagrant deployment. Setting an `artifacts_dir` variable here to "./" should have Kubespray copy the config into the current dir.

Or you could just drop this variable all together.

Reply:
Setting `artifacts_dir: './'` causes the artifacts to be dumped in the kubespray directory, so lets just remove `kubeconfig_localhost`.